ts/pages/staking/home.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+import { colors } from 'ts/style/colors';
+
+import { Button } from 'ts/components/button';
+import { CFLMetrics } from 'ts/pages/cfl/cfl_metrics';
+
+import { ConnectWalletDialog } from 'ts/components/dialogs/connect_wallet_dialog';
+import { StakingConfirmationDialog } from 'ts/components/dialogs/staking_confirmation_dialog';
+import { StakingPageLayout } from 'ts/components/staking/layout/staking_page_layout';
+import { StakingPoolDetailRow } from 'ts/components/staking/staking_pool_detail_row';
+
+import { ScreenWidths } from 'ts/types';
+
+import { StakingHero } from 'ts/components/staking/hero';
+import { Heading } from 'ts/components/text';
+
+const stakingPools = [
+  {
+    id: '29n5c290cn0cc2943cn239',
+    name: 'Staking 01',
+    thumbnailUrl: 'https://cdn.worldvectorlogo.com/logos/0x-virtual-money-.svg',
+    feesCollectedEth: 0.03281,
+    stakingPercent: 2000,
+    rewardsSharePercent: 100,
+    location: '234 CALIFORNIA',
+  },
+  {
+    id: '29n5c290cn0cc2943cn230',
+    name: 'Staking 02',
+    thumbnailUrl: 'https://cdn.worldvectorlogo.com/logos/0x-virtual-money-.svg',
+    feesCollectedEth: 0.03281,
+    stakingPercent: 2000,
+    rewardsSharePercent: 100,
+    location: '234 CALIFORNIA',
+  },
+  {
+    id: '29n5c290cn0cc2943cn240',
+    name: 'Staking 03',
+    thumbnailUrl: 'https://cdn.worldvectorlogo.com/logos/0x-virtual-money-.svg',
```

Comment:
is this asset included in the PR? ideally it would be just `/logos/0x-virtual-money-svg`

Reply:
We assumed all the data including icons gets fetched from the API (in case more staking pools get added). This is just some ‘mock’ data. Are we right in our assumption?